solc/CommandLineInterface.cpp

```diff
@@ -790,7 +799,7 @@ void CommandLineInterface::handleCombinedJSON()
 	{
 		Json::Value contractData(Json::objectValue);
 		if (requests.count(g_strAbi))
-			contractData[g_strAbi] = dev::jsonCompactPrint(m_compiler->contractABI(contractName));
```

Comment:
I still think this and the natspec fields cannot be changed from serialised in a regular release, though it should be  fine adding the pretty print as an option here. We're aiming to another breaking release (0.5.0) soon, which could then include this part of the change.

What do you think?